src/api.jsx

```diff
@@ -1,16 +1,40 @@
-import axios from 'axios'; 
-import useGoogleSheets from 'use-google-sheets';
-
-const SPREADSHEET_ID = '1gr-fnI5u6pvbTBpvnPQ5vKgabpvBKyDho8wCCk8C0yI'; 
-const API_KEY = 'AIzaSyCa44AD3zAObDSuW6NFfPW_SrgO7pOH9G0'; 
+import axios from 'axios';
+import { fetchGoogleSheetsData } from 'google-sheets-mapper'
 
 //google spreadsheet api methods
-export async function getFacilitatorPics() {
-   const { data, loading, error } = useGoogleSheets({
-      apiKey: API_KEY,
-      sheetId: SPREADSHEET_ID,
-      sheetsOptions: [{ id: 'Facilitator Pics' }]
-    });
+export const getSheet = async ( sheetName ) => {
+   try {
+      return await fetchGoogleSheetsData({
+         apiKey: import.meta.env.VITE_GOOGLE_API_KEY,
+         sheetId: import.meta.env.VITE_SPREADSHEET_ID,
+         sheetsOptions: [{ id: sheetName }],
+      });
+   } catch (error) {
+      console.erorr(error);
+      throw error;
+   }
+}
+
+export const getLinks = async ( linkNames ) => {
+   try {
+      const incomingData = await fetchGoogleSheetsData({
+         apiKey: import.meta.env.VITE_GOOGLE_API_KEY,
+         sheetId: import.meta.env.VITE_SPREADSHEET_ID,
+         sheetsOptions: [{ id: 'Links' }],
+      });
+
+      let tempData = incomingData[0].data[0];
+      const links = {};
+
+      linkNames.forEach(name => {
+         links[name] = tempData[name];
+         console.log(name);
+      });
+
+      console.log("api.jsx#getLinks: requested links: ", links);
```

Comment:
this function is beautiful and this might be super picky but can you remove this `console.log`?